CosmosContracts/juno-tools
# feat: improve meta tags and favicon assets

Fixes #9

## Description

This PR updates the base favicon and generates various assets for proper SEO.

## Changes

List any technical changes.

- [x] add npm script to generate favicons
- [x] add icon.png
- [x] generate favicons assets to `/public/assets/`
- [x] remove unused assets

## Screenshots

TODO

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- open any page and make sure the tab icon is updated

## Links

\-
